Fix following error with GCC-4.3.3:
/home/peter/work/logic/svn/eps_svn/software/products/linux/LTIB/trunk/ltib-20091102/rpm/BUILD/qtopia-4-4.3.0/qtopia-opensource-4.3.0/qtopiacore/qt/src/gui/painting/qdrawhelper.cpp:5737: error: explicit template specialization cannot have a storage class
/home/peter/work/logic/svn/eps_svn/software/products/linux/LTIB/trunk/ltib-20091102/rpm/BUILD/qtopia-4-4.3.0/qtopia-opensource-4.3.0/qtopiacore/qt/src/gui/painting/qdrawhelper.cpp:5761: error: explicit template specialization cannot have a storage class


diff --exclude CVS --exclude .git -uNr a/qtopiacore/qt/src/gui/painting/qdrawhelper.cpp b/qtopiacore/qt/src/gui/painting/qdrawhelper.cpp
--- a/qtopiacore/qt/src/gui/painting/qdrawhelper.cpp	2007-10-29 03:14:25.000000000 -0400
+++ src/gui/painting/qdrawhelper.cpp	2009-11-02 16:42:17.000000000 -0500
@@ -5715,7 +5715,7 @@
 
 #ifdef QT_QWS_DEPTH_24
 template <>
-static inline void qt_memrotate90_template<quint24, quint32>(const quint32 *src,
+inline void qt_memrotate90_template<quint24, quint32>(const quint32 *src,
                                                              int srcWidth, int srcHeight, int srcStride,
                                                              quint24 *dest, int dstStride)
 {
@@ -5739,7 +5739,7 @@
 
 #ifdef QT_QWS_DEPTH_18
 template <>
-static inline void qt_memrotate90_template<quint18, quint32>(const quint32 *src,
+inline void qt_memrotate90_template<quint18, quint32>(const quint32 *src,
                                                              int srcWidth, int srcHeight, int srcStride,
                                                              quint18 *dest, int dstStride)
 {
 
